js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/util.tsx

```diff
@@ -15,3 +20,104 @@ export function nodePathKey(node: {path: string; id: string} | {id: string}) {
 export function getDisplayName(node: GraphNode) {
   return node.assetKey.path[node.assetKey.path.length - 1]!;
 }
+
+export function StatusCaseDot({statusCase}: {statusCase: StatusCase}) {
```

Comment:
Might be nice to repurpose https://dagit-core-storybook.vercel.app/?path=/docs/runstatus--docs for these.